libraries/platforms/bigblade-verilator/bsg_manycore_platform.cpp

```diff
@@ -162,6 +162,15 @@ void hb_mc_platform_cleanup(hb_mc_manycore_t *mc)
         auto key = active_ids.find(platform->id);
         active_ids.erase(key);
 
+        delete platform->top;
+        auto m = machines.find(platform->id);
```

Comment:
When are there more than one machine?

Reply:
This is legacy, but hasn't been removed. When I wrote it, I was trying to support having having multiple HB instances being simulated.

Reply:
Okay!